vision_datasets/common/data_manifest.py

```diff
@@ -447,6 +451,17 @@ def create_dataset_manifest(coco_file_path_or_url, data_type, container_sas_or_r
 
         file_reader.close()
 
+        if data_type == DatasetTypes.IMCAP:
+            images_by_id = {}
+            for img in coco_manifest['images']:
+                label = {'image_id': img['id'], 'file_name': img['file_name'], 'caption': []}
```

Comment:
why do we want to save the image id and file_name again in the label?

Reply:
This 'image_id' and 'file_name' information is used to construct the prediction json file in the inference step, so that we can use pycocoevalcap to evaluate the result

Reply:
ImageDataManifest has those info. why save it again in the label?